wix-pricing-plans-v2/wix-pricing-plans-v2/Orders.service.json

```diff
@@ -490,26 +1567,37 @@
         ],
         "examples": [
           {
-            "title": "markAsPaid example",
+            "title": "Mark an order as paid ",
             "body": [
               "import { orders } from 'wix-pricing-plans.v2';",
-              "  ",
-              " async function markAsPaid(id) {",
-              "   try {",
-              "     const result = await orders.markAsPaid(id);",
+              "import { elevate } from 'wix-auth';",
               "",
-              "     return result;",
-              "   } catch (error) {",
-              "     console.error(error);",
-              "     // Handle the error",
-              "   }",
-              " }",
-              "  "
+              "/* Sample _id value: '82d99338-5653-459a-a751-b57483f7cfb5' */",
+              "",
+              "const elevatedMarkAsPaid = elevate(orders.markAsPaid);",
+              "",
+              "export async function myMarkAsPaidFunction(_id) {",
+              "  try {",
+              "    await elevatedMarkAsPaid(_id);",
+              "",
+              "    return;",
```

Comment:
I'm guessing we don't return anything or set elevatedMarkAsPaid to a variable because the promise is void? Is that a new style guideline?

Reply:
Yes